sfm_pc/management/commands/import_google_doc.py

```diff
@@ -672,242 +672,243 @@ def create_organization(self, org_data):
+                                },
+                            }
+
+                            try:
+                                composition = Composition.objects.get(compositionparent__value=parent_organization,
+                                                                      compositionchild__value=organization)
+                            except Composition.DoesNotExist:
+                                composition = Composition.create(comp_info)
+
+                            self.make_relation('StartDate',
+                                               composition_positions['StartDate'],
+                                               org_data,
+                                               composition,
+                                               date=True)
+
+                            self.make_relation('EndDate',
+                                               composition_positions['EndDate'],
+                                               org_data,
+                                               composition,
+                                               date=True)
+
+                            self.make_real_date(data=org_data,
+                                                position=composition_positions['RealStart']['value'],
+                                                model=CompositionRealStart,
+                                                attribute='realstart',
+                                                object_ref=composition)
+
+                            self.make_relation('OpenEnded',
+                                               composition_positions['OpenEnded'],
+                                               org_data,
+                                               composition)
+
+                            self.make_relation('Classification',
+                                               composition_positions['Classification'],
+                                               org_data,
+                                               composition)
 
-                # Make memberships
 
-                try:
-                    member_org_name = org_data[membership_positions['OrganizationOrganization']['value']]
-                except IndexError:
-                    member_org_name = None
+                        else:
+                            self.log_error('Parent organization for {} does not have source or confidence'.format(organization.name))
 
-                if member_org_name:
-                    try:
-                        confidence = self.get_confidence(org_data[membership_positions['OrganizationOrganization']['confidence']])
-                    except (IndexError, KeyError):
-                        self.log_error('Member organization for {} does not have confidence'.format(member_org_name))
-                        confidence = None
+                elif org_data['unit:relation_type'] == 'member':
```

Comment:
Other half of the switch. Used `elif` instead of `else` because `unit:relation_type` can also be null, in which case neither a composition nor a membership should be created.